samza-core/src/test/java/org/apache/samza/startpoint/TestStartpointManager.java

```diff
@@ -170,21 +170,6 @@ public void testBasics() {
     startpointManager.stop();
   }
 
-  @Test
-  public void testStaleStartpoints() {
-    SystemStreamPartition ssp = new SystemStreamPartition("mockSystem", "mockStream", new Partition(2));
-    TaskName taskName = new TaskName("MockTask");
-
-    startpointManager.start();
-    long staleTimestamp = Instant.now().toEpochMilli() - StartpointManager.DEFAULT_EXPIRATION_DURATION.toMillis() - 2;
-    MockStartpointCustom startpoint = new MockStartpointCustom("das boot", 42, staleTimestamp);
```

Comment:
Did this test specifically need `StartpointCustom`? It doesn't seem like it needs to be fully removed.